### PR TITLE
Link members and repositories count to each page on org home.

### PR DESCRIPTION
**Pull request by harryxu**
As title, for save clicks to org's members and repositories page.

### CODE REVIEW

**Comment by harryxu** [comment]
Add screenshot for demonstration.

febcc303-2595-416d-9223-b12db5c7d3fe

File: templates/org/home.tmpl [hunk 1]
```diff
@@ -64,7 +64,10 @@
 						{{range .Teams}}
 							<div class="item">
 								<a href="{{$.OrgLink}}/teams/{{.LowerName}}"><strong class="team-name">{{.Name}}</strong></a>
-								<p class="text grey">{{.NumMembers}} {{$.i18n.Tr "org.lower_members"}} · {{.NumRepos}} {{$.i18n.Tr "org.lower_repositories"}}</p>
+								<p class="text grey">
+									<a href="{{$.OrgLink}}/teams/{{.LowerName}}"><strong>{{.NumMembers}}</strong> {{$.i18n.Tr "org.lower_members"}}</a> ·
+									<a href="{{$.OrgLink}}/teams/{{.LowerName}}/repositories"><strong>{{.NumRepos}}</strong> {{$.i18n.Tr "org.lower_repositories"}}</a>
```
Comment (Morlinest) [inline templates/org/home.tmpl:69]: Looks good but I am not sure about adding `<strong>`. For me it doesn't look nice.

Comment (lafriks) [reply templates/org/home.tmpl:69]: I think bold is ok

**Comment by lafriks** [comment]
LGTM

**Comment by Morlinest** [comment]
LGTM

**Pull request merged**